docs/modules/operation/pages/deep-dive/service-assurance/scheduled-outages.adoc

```diff
@@ -0,0 +1,89 @@
+
+= Scheduled Outages
+
+Scheduled outages let you suppress notifications and pause status polling, threshold checking, and data collection during a specified time period.
+You might want to schedule an outage for any of the following reasons:
+
+* Updating components.
+* Completing scheduled reboots.
+* Planned decommissioning of nodes.
+* Preventing excessive notifications to on-call staff.
+* Completing scheduled infrastructure testing.
+* Other situations where known outages may be caused or thresholds may be reached
+
+When you define a scheduled outage, you must set the following options:
+
+* Name.
+* Nodes to include.
+* Node interfaces to include.
+* Outage type (specific, daily, weekly, monthly).
+* Date and time for the outage.
+* Suppression settings (all notifications, status polling, threshold checking, data collection).
+
+[caption=]
```

Comment:
I don't know that the "default value" column is needed.  Only the Outage Type field has a default, and because it shows in the Web UI, it probably doesn't need to be called out there.